tools/autograd/derivatives.yaml

```diff
@@ -1088,11 +1088,11 @@
 - name: squeeze_.dim(Tensor(a!) self, int dim) -> Tensor(a!)
   self: unsqueeze_to(grad, dim, self.sizes())
 
-- name: std(Tensor self, bool unbiased=True) -> Tensor
-  self: std_backward(result, grad, self, unbiased)
+- name: std.correction(Tensor self, int[1]? dim, *, int? correction, bool keepdim=False) -> Tensor
```

Comment:
I'm curious if `int? correction` should be the thing here?

Are std and var going to be implemented so they call into std(..., correction=...) and var(..., corection-...)? If so, should correction be optional here?

Reply:
> Are std and var going to be implemented so they call into std(..., correction=...) and var(..., correction-...)?

Yes, the `unbiased` overloads both convert to numerical correction and call the correction overload.

>If so, should correction be optional here?

It has to exactly match the JIT schema in native_functions.yaml